### PR TITLE
OpenGL changes:

### PR DESCRIPTION
**Pull request by udhos**
- Make setRotationMatrix() compatible, in respect to inversion, with setViewMatrix().
- Add helper setModelMatrix().
- Add slightly simpler/faster setViewMatrix2().

### CODE REVIEW

File: lib/src/vector_math/opengl.dart [hunk 1]
```diff
@@ -59,12 +59,64 @@ part of vector_math;
  * 
  * }
  */
-void setRotationMatrix(
-    Matrix4 rotationMatrix, Vector3 forwardDirection, upDirection) {
+void setRotationMatrix(Matrix4 rotationMatrix, Vector3 forwardDirection,
+                       Vector3 upDirection) {
```
Comment (johnmccutchan) [inline lib/src/vector_math/opengl.dart]: fix formatting

File: lib/src/vector_math/opengl.dart [hunk 2]
```diff
@@ -59,12 +59,64 @@ part of vector_math;
...
+    c3[0], c3[1], c3[2], 0.0,
+    tx, ty, tz, 1.0);
+}
+
+/**
+ * Constructs an OpenGL view matrix in [viewMatrix].
+ * View transformation is the inverse of the model transformation.
+ * View matrix is commonly used to compute the camera location/orientation into the full model-view stack.
+ * 
+ * Sligthly simpler/faster implementation for setViewMatrix() below.
```
Comment (johnmccutchan) [inline lib/src/vector_math/opengl.dart]: Should this just replace setViewMatrix?

**Comment by udhos** [comment]
John,
1) fixed the formatting, but the formatter affected other lines as well.
2) replaced setViewMatrix() with the new version.
Thanks,
Everton

**Comment by udhos** [comment]
Please take another look at this request.

**Pull request merged**